src/gui/accountsettings.cpp

```diff
@@ -339,21 +342,33 @@ void AccountSettings::slotMarkSubfolderEncrypted(FolderStatusModel::SubFolderInf
 
     const auto folder = folderInfo->_folder;
     Q_ASSERT(folder);
+    const auto path = folderInfo->_path;
+    const auto fileId = folderInfo->_fileId;
+    const auto encryptFolder = [this, fileId, folder, path] {
+        // Folder info have directory paths in Foo/Bar/ convention...
+        Q_ASSERT(!path.startsWith('/') && path.endsWith('/'));
+        // But EncryptFolderJob expects directory path Foo/Bar convention
+        const auto choppedPath = path.chopped(1);
+
+        // Does the file still exist?
+        SyncJournalFileRecord record;
+        if (!folder->journalDb()->getFileRecord(choppedPath, &record) || !record.isValid()) {
+            return;
```

Comment:
@FlexW maybe, a warning needs to be here? Just to make sure we can track issues when someone reports to us that they can't encrypt a folder?